include/deal.II/lac/chunk_sparsity_pattern.h

```diff
@@ -433,13 +443,12 @@ class ChunkSparsityPattern : public Subscriptor
    * these iterators) must be a container itself that provides functions
    * <tt>begin</tt> and <tt>end</tt> designating a range of iterators that
    * describe the contents of one line. Dereferencing these inner iterators
-   * must either yield a pair of an unsigned integer as column index and a
+   * must either yield a pair of a scalar as column index and a
```

Comment:
Do we use `scalar` in other contexts like this? I would have preferred `integral number` or simply `integer` (without the `unsigned` qualifier to avoid indicating the exact type here which does not matter).